### PR TITLE
Update release notes the overview and other pages

### PR DESCRIPTION
**Pull request by mmattel**
This update:

* Move content to the correct location
* Correct release notes + add breaking changes for ocis
* Improve the overview section

Language review welcomed

@EParzefall pls do a local build and check the links changed, 4 eyes is better than 2 😄

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -2,33 +2,197 @@
...
+The new iOS client version 12 also comes with full support for Infinite Scale spaces and allows users comfortable access to the collaboration spaces in all their accounts. The list of spaces available is dynamically retrieved from the Infinite Scale server and listed along with the cover image and description. 
+
+The iOS client is an example of how efficient data management works on the mobile platform with Infinite Scale.
+////
+
+=== Notes on Native Client Support for Spaces
+
+The latest Infinite Scale features are supported since ownCloud Desktop release 3.0.
+
+The iOS and Android apps do currently only support the home space.
```
Comment (phil-davis) [inline modules/ROOT/pages/index.adoc:72]:
```suggestion
The iOS and Android apps currently only support the home space.
```

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -2,33 +2,197 @@
...
+
+| Sharing / Access management (permissions on data)
+| Ability for users to share files and folders with users, groups and via links
+
+| Spaces
+| User-independent project and team folders with multiple owners and dedicated behavior
+
+| Deleted Files / Trash bin
+| Ability for users to restore deleted files
+
```
Comment (phil-davis) [inline modules/ROOT/pages/index.adoc:122]: Extra line.

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -2,33 +2,197 @@
...
+| Ability for users to download multiple files and folders as archives
+
+.5+| Platform
+| User settings
+| Framework for users to define their settings like web UI language
+
+| Notifications
+| Framework to inform users about events in the platform; email notification for the event “Share received”
+
+| File Search
```
Comment (EParzefall) [inline modules/ROOT/pages/index.adoc:143]:
```suggestion
| File search
```

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -2,33 +2,197 @@
...
+
+| Auditing
+| Ability to log all operations conducted by users for compliance and traceability
+
+.4+| Deployment
+| Single binary
+| The simplest way to deploy Infinite Scale for testing
+
+| Systemd
+| Deploy Infinite Scale as a Systemd service
```
Comment (EParzefall) [inline modules/ROOT/pages/index.adoc:181]:
```suggestion
| Deploy Infinite Scale as a systemd service
```

File: modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -2,33 +2,197 @@
...
+| Technical log for operations and debugging
+
+| Auditing
+| Ability to log all operations conducted by users for compliance and traceability
+
+.4+| Deployment
+| Single binary
+| The simplest way to deploy Infinite Scale for testing
+
+| Systemd
```
Comment (EParzefall) [inline modules/ROOT/pages/index.adoc:180]:
```suggestion
| systemd
```

File: modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -2,33 +2,197 @@
...
+| Deploy Infinite Scale as a Systemd service
+
+| Docker
+| Deploy Infinite Scale using Docker and Docker-Compose
+
+| Helm / Kubernetes
+| Deploy Infinite Scale in Kubernetes
+
+.3+| Integrations
```
Comment (EParzefall) [inline modules/ROOT/pages/index.adoc:189]:
```suggestion
.3+| Web Office Integrations
```

File: modules/ROOT/pages/index.adoc [hunk 7]
```diff
@@ -2,33 +2,197 @@
...
+| Web Office: Collabora Online
```
Comment (EParzefall) [inline modules/ROOT/pages/index.adoc:190]:
```suggestion
| Collabora Online
```

File: modules/ROOT/pages/index.adoc [hunk 8]
```diff
@@ -2,33 +2,197 @@
...
+| Collabora Online can be integrated via the wopiserver extension; automatic file locking capabilities are available
+
+| Web Office: ONLYOFFICE
```
Comment (EParzefall) [inline modules/ROOT/pages/index.adoc:193]:
```suggestion
| ONLYOFFICE
```

File: modules/ROOT/pages/index.adoc [hunk 9]
```diff
@@ -2,33 +2,197 @@
...
+| ONLYOFFICE can be integrated via the wopiserver extension; automatic file locking capabilities are available
+
+| Web Office: Microsoft Office Online
```
Comment (EParzefall) [inline modules/ROOT/pages/index.adoc:196]:
```suggestion
| Microsoft Office Online
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 10]
```diff
@@ -15,145 +15,36 @@ toc::[]
...
 
-=== What is ownCloud?
-ownCloud is an open source server application that allows you to access your files from anywhere in a secure way. 
-The files are stored on a server running ownCloud. You can access your files via the browser or sync them to your desktop or mobile device like you might know it from oneDrive, Dropbox or others. The difference with ownCloud is, that you stay in control of your data as you can install ownCloud in your own environment.
+Please find the full list of changes here:
 
-=== What is Infinite Scale?
-Infinite Scale is the latest generation of the ownCloud server application.
-With its modern architecture, Infinite Scale provides all deployment models for cloud infrastructure deployments and optimized scaling setups for the best output for the invested energy.
+* ocis: https://github.com/owncloud/ocis/releases/tag/v2.0.0[Changes in 2.0.0]
```
Comment (EParzefall) [inline modules/ROOT/pages/ocis_release_notes.adoc:22]:
```suggestion
* Infinite Scale: https://github.com/owncloud/ocis/releases/tag/v2.0.0[Changes in 2.0.0]
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 11]
```diff
@@ -15,145 +15,36 @@ toc::[]
...
+* web: https://github.com/owncloud/web/releases/tag/v6.0.0[Changelog for ownCloud Web 6.0.0]
```
Comment (EParzefall) [inline modules/ROOT/pages/ocis_release_notes.adoc:23]:
```suggestion
* Web: https://github.com/owncloud/web/releases/tag/v6.0.0[Changelog for ownCloud Web 6.0.0]
```

**Review by EParzefall — APPROVED** [review]

**Pull request merged**